### PR TITLE
[#16] Create test (story)

### PR DESCRIPTION
**Pull request by alexDose**

### CODE REVIEW

File: src/Types/NewTestTypes.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {SelectChangeEvent} from "@mui/material/Select";
```
Comment (gitkoshelew) [inline src/Types/NewTestTypes.ts]: Why you name file NewTestTypes are types different from some regular Test ?

File: src/assets/icons/ArrowBottomIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const ArrowBottomIcon = () => {
+    return (
+        <svg width="14" height="9" viewBox="0 0 14 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M1.06661 0.80994L6.99963 6.87618L13.0659 0.943152" stroke="black" stroke-opacity="0.3" stroke-width="2"/>
```
Comment (EvelinSu) [inline src/assets/icons/ArrowBottomIcon.tsx]: Rename attributes in all icons on camelcase (stroke-opacity must be strokeOpacity etc)

File: src/components/new-test/ButtonSaveTest.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export const ButtonSaveTest = () => {
+
+    return <Stack alignItems={'center'}>
+        <Button variant={'contained'}
+                sx={{
+                    width: '132px',
+                    height: '42px',
+                    textTransform: 'inherit'
+                }}>
```
Comment (EvelinSu) [inline src/components/new-test/ButtonSaveTest.tsx]: (line 7-11) It's all in the theme, just delete sx and variant

File: src/components/new-test/SuperSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            color: 'primary.contrastText',
+            borderRadius: '15px',
+            typography: 'subtitle1',
+            '& .MuiOutlinedInput-notchedOutline': {
+                border: 0
+            },
+            '& .MuiSvgIcon-root': {
+                color: 'text.secondaryAlpha500'
+            },
+        }}
```
Comment (EvelinSu) [inline src/components/new-test/SuperSelect.tsx]: This is the default select for our site, you need to add it to the theme (in this case select on home page must be secondary, that is, you need to rewrite the styles in the theme specifically for secondary)

**Review by gitkoshelew — APPROVED** [review]

**Review by EvelinSu — APPROVED** [review]

**Pull request merged**